### ISSUE TITLE
No Audio iOS

### ISSUE THREAD

**Issue by pmwisdom** [floatinghotpot/cordova-plugin-nativeaudio#47]
So my problem is confusing. 

I preload a complex file, which succeeds. And then I play the file I loaded, which also succeeds, and then completes but no sound actually plays, I can only tell via the success / complete callbacks that it actually completes. It works find on android. It's almost like its treating it like its muted.

**Comment by pmwisdom** [comment]
Found out why, had the cordova media plugin installed as well as this one. After I played a sound through the cdv media plugin, the native audio plugin started playing its sounds. Weird.

**Comment by pmwisdom** [comment]
I put this in pluginInitialize and it started working:

````
[session setCategory:AVAudioSessionCategoryPlayback error:nil];
````

**Comment by michaelkariv** [comment]
Is it possible to update the version of this plugin on phonegap build ? I depend on them for builds and they have the old version where preloadComplex does not play.

**Comment by floatinghotpot** [comment]
it's published to npm just now as v3.0.6, if you mean that.

**Comment by midasdas** [comment]
It's still broken on PhoneGap Build. `preloadComplex` doesn't work at all, on iOS nor Android. It sends success messages but no audio is played.

Any ideas for a fix? Thanks
